test/relayers/pimlico.test.ts

```diff
@@ -1,70 +1,73 @@
-import { Chain } from '@parifi/references';
```

Comment:
Pimlico is still relevant with the new integration. Please uncomment and test everything related to it